src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter.Integer.cs

```diff
@@ -64,7 +64,7 @@ static bool TryFormatCore(long value, ulong mask, Span<byte> buffer, out int byt
             else if (encoder.IsInvariantUtf16)
                 return TryFormatInvariantUtf16(value, mask, buffer, out bytesWritten, format);
             else
-                throw new NotImplementedException();
+                return IntegerFormatter.TryFormatInt64(value, mask, buffer, out bytesWritten, format, encoder);
```

Comment:
The globalization formatting code does. It isn't a path we care about until we iron out the localization / globalization better.